tests/test_rpm_verifier.py

```diff
@@ -15,178 +15,164 @@
                 """
-                Found unsigned RPMs:
-                i1: my-rpm
+                libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890
+                python39-twisted-23.10.0-1.el8ap (none)
+                libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890
+                gpg-pubkey-d4082792-5b32db75 (none)
                 """
             ).strip(),
-            True,
-            id="Unsigned RPM + no errors. Report failures",
-        ),
-        pytest.param(
             [
-                ProcessedImage(image="i1", unsigned_rpms=["my-rpm", "another-rpm"]),
-                ProcessedImage(image="i2", unsigned_rpms=[]),
+                "libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "python39-twisted-23.10.0-1.el8ap (none)",
+                "libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
+                "gpg-pubkey-d4082792-5b32db75 (none)",
             ],
-            dedent(
-                """
-                Found unsigned RPMs:
-                i1: my-rpm, another-rpm
-                """
-            ).strip(),
-            True,
-            id="An image with multiple unsigned and another without. Report failure",
+            id="Mix of signed and unsigned",
         ),
         pytest.param(
-            [
-                ProcessedImage(
-                    image="i1", unsigned_rpms=[], error="Failed to run command"
-                ),
-                ProcessedImage(image="i2", unsigned_rpms=["their-rpm"]),
-            ],
             dedent(
                 """
-                Found unsigned RPMs:
-                i2: their-rpm
-                Encountered errors:
-                i1: Failed to run command 
+                libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890
+                libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890
                 """
             ).strip(),
-            True,
-            id="Error in running command + image with unsigned RPMs. Report failure",
-        ),
-        pytest.param(
             [
-                ProcessedImage(
-                    image="i1", unsigned_rpms=[], error="Failed to run first command"
-                ),
-                ProcessedImage(
-                    image="i2", unsigned_rpms=[], error="Failed to run second command"
-                ),
+                "libssh-config-0.9.6-10.el8_8 RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "libmodulemd-2.13.0-1.el8 RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
             ],
-            dedent(
-                """
-                Encountered errors:
-                i1: Failed to run first command
-                i2: Failed to run second command
-                """
-            ).strip(),
-            True,
-            id="Multiple errors in running command + no unsigned RPMs. Report failure",
+            id="All signed",
```

Comment:
As long as we're just testing that we're returning whatever the runner returns, I don't think there's an added value in differentiating between All signed or mixed inputs.

Reply:
This makes me wonder whether the `get_rpms_data` function is actually needed.

Reply:
Maybe we can assert on the list of calls to the runner to make sure the input is composed correctly.